_common/qiskit/benchmark_runner.py

```diff
@@ -88,6 +93,26 @@
 
     args = parser.parse_args()
 
+# For Inserting the Noise model default into exec option as it is function call 
+    if args.noise_model != None :
+        
+        if args.noise_model == 'None':
+           args.exec_options["noise_model"] = None 
+        else:
+            module,method = args.noise_model.split(".")
+            module = globals()[module]
+            method = method.split("(")[0]
+            custom_noise = getattr(module, method)
+            print(custom_noise)
+            noise = custom_noise()
+            args.exec_options["noise_model"] = noise
+    
+    # Provdider detail update using provider module name and class name
+    if args.provider_module_name != None and args.provider_class_name != None:
```

Comment:
When checking whether something is `None` is Python, we never want to use `!=` or `==` equality operators, instead, we want to use `is not` and `is`, respectively.